src/collections/_documentation/learn/configuration.md

```diff
@@ -0,0 +1,204 @@
+
+{:.config-key}
+### send-default-pii
+
+If this flag is enabled then certain personally identifyable information is added by active
+integrations.  Without this flag they are never added to the event to begin with.  If possible
+it's recommended to turn on this feature and use the server side PII stripping to remove the
+values instead.
+
+{:.config-key}
+### server-name
+
+{% unsupported browser %}
+Can be used to supply a "server name".  When provided the name of the server is sent along and
+persisted in the event.  Note that for many integrations the server name actually corresponds to
+the device hostname even in situations where the machine is not actually a server.  Most SDKs
+will attempt to auto discover this value.
+{% endunsupported %}
+
+{:.config-key}
+### blacklist-urls
+
+{% supported browser %}
+A pattern for error URLs which should not be sent to Sentry.  To whitelist certain errors instead,
+use `whitelist-urls`.  By default, all errors will be sent.  This only applies to events sent
+from browsers.
+{% endsupported %}
+
+{:.config-key}
+### whitelist-urls
+
+{% supported browser %}
+A pattern for error URLs which should exclusively be sent to Sentry.  To
+blacklist certain errors instead, use `blacklist-urls`.  By default, all errors
+will be sent.  This only applies to events sent from browsers.
+{% endsupported %}
+
+{:.config-key}
+### request-bodies
+
+{% supported python %}
+This parameter controls if integrations should capture HTTP request bodies.  It can be set to one
+of the following values:
+
+* `never`: request bodies are never sent.
+* `small`: only small request bodies will be captured where the cutoff for small depends on the SDK (typically 4KB)
+* `medium`: medium sized requests and small requests will be captured. (typically 10KB)
+* `always`: the SDK will always capture the request body for as long as sentry can make sense of it
+{% endsupported %}
+
+{:.config-key}
+### with-locals
+
+{% supported python %}
+When enabled local variables are sent along with stackframes.  This can have a performance
+and PII impact.  Enabled by default on platforms where this is available.
+{% endsupported %}
+
+## Integration Configuration
+
+For many platform SDKs integrations can be configured alongside it.  On some platforms that
+happens as part of the `init()` call, in some others different patterns apply.
+
+{:.config-key}
+### integrations
+
+{% unsupported csharp aspnetcore rust %}
+In some SDKs the integrations are configured through this parameter on library initialization.
+For more information have a look at the specific integration documentation.
+{% endunsupported %}
+
+{:.config-key}
+### default-integrations
+
+{% supported python %}
+This can be used to disable integrations that are added by default.  When set to `false` no
+default integrations are added.
+{% endsupported %}
+
+## Hooks
+
+These options can be used to hook the SDK in various ways to customize the reporting of events.
```

Comment:
This and the `Transport Options` are technically a subcategory of `Common Options`, right?